src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4302,596 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <value>Template Info</value>
+    </labels>
+    <labels>
+        <fullName>geToastTemplateCreateSuccess</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Template successfully created</shortDescription>
+        <!-- {0} is a string. Name of the template that was created. -->
+        <value>The {0} template was created.</value>
+    </labels>
+    <labels>
+        <fullName>geToastSaveFailed</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when trying to save if there isn't a system error to display</shortDescription>
+        <value>Your changes cannot be saved. Please try again.</value>
+    </labels>
+    <labels>
+        <fullName>geToastSelectActiveSection</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning when attempting to add a gift field and a section hasn't been selected.</shortDescription>
+        <value>Please select a section.</value>
+    </labels>
+    <labels>
+        <fullName>geToastTemplateTabsError</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when trying to save and tabs has an error</shortDescription>
+        <!-- {0} is a list of strings. Names of the tabs with errors. -->
+        <value>Please review tabs {0}</value>
+    </labels>
+    <labels>
+        <fullName>geToastTemplateTabError</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when trying to save and a tab has an error</shortDescription>
+        <!-- {0} is a string. Name of the tab with errors. -->
+        <value>Please review tab {0}</value>
+    </labels>
+    <labels>
+        <fullName>geToastTemplateUpdateSuccess</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Template successfully updated</shortDescription>
+        <!-- {0} is a string. Name of the template that was created. -->
+        <value>The {0} template was updated.</value>
+    </labels>
+    <labels>
+        <fullName>geWarningFormFieldsModalDeleteSection</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning message for deleting a section if it contains required field</shortDescription>
+        <value>Warning: By deleting this section, you are also removing required fields from the template. These fields need to be added into another section before you can save the template.</value>
```

Comment:
@psadie-sfdo does this warning show up in red text? I think that's how we had it in the original labels spreadsheet and that formatting got lost with the move to Google Sheets.

Reply:
@melissabarber - Not in red. This is a warning and is wrapped in a yellow notification card.
<img width="742" alt="Screen Shot 2019-12-05 at 1 11 33 PM" src="https://user-images.githubusercontent.com/25848103/70748829-a3d2b000-1cdf-11ea-827f-ae3d7f11382f.png">